src/main/kotlin/com/terraformation/backend/accelerator/variables/AcceleratorProjectVariableValuesService.kt

```diff
@@ -0,0 +1,340 @@
+package com.terraformation.backend.accelerator.variables
+
+import com.terraformation.backend.accelerator.model.ProjectAcceleratorVariableValuesModel
+import com.terraformation.backend.customer.model.SystemUser
+import com.terraformation.backend.customer.model.requirePermissions
+import com.terraformation.backend.db.default_schema.LandUseModelType
+import com.terraformation.backend.db.default_schema.ProjectId
+import com.terraformation.backend.db.default_schema.tables.daos.CountriesDao
+import com.terraformation.backend.db.docprod.VariableId
+import com.terraformation.backend.documentproducer.db.VariableStore
+import com.terraformation.backend.documentproducer.db.VariableValueStore
+import com.terraformation.backend.documentproducer.model.ExistingNumberValue
+import com.terraformation.backend.documentproducer.model.ExistingSelectValue
+import com.terraformation.backend.documentproducer.model.ExistingTextValue
+import com.terraformation.backend.documentproducer.model.ExistingValue
+import com.terraformation.backend.documentproducer.model.NumberVariable
+import com.terraformation.backend.documentproducer.model.SelectVariable
+import com.terraformation.backend.documentproducer.model.TextVariable
+import com.terraformation.backend.documentproducer.model.ValueOperation
+import com.terraformation.backend.documentproducer.model.Variable
+import com.terraformation.backend.log.perClassLogger
+import jakarta.inject.Named
+
+@Named
+class AcceleratorProjectVariableValuesService(
```

Comment:
Minor naming thing: this class's name uses the term "accelerator project variable" whereas the model class's name uses "project accelerator variable." Any reason those shouldn't be consistent?

Reply:
standardized all to "projectAccelerator"